code/core/package.json

```diff
@@ -95,48 +80,32 @@
     },
     "./internal/common": {
       "types": "./dist/common/index.d.ts",
-      "import": "./dist/common/index.js",
       "default": "./dist/common/index.js"
     },
     "./internal/components": {
       "types": "./dist/components/index.d.ts",
-      "import": "./dist/components/index.js",
-      "require": "./dist/components/index.cjs"
+      "default": "./dist/components/index.js"
     },
     "./internal/core-events": {
       "types": "./dist/core-events/index.d.ts",
-      "import": "./dist/core-events/index.js",
-      "require": "./dist/core-events/index.cjs"
+      "default": "./dist/core-events/index.js"
     },
     "./internal/core-server": {
       "types": "./dist/core-server/index.d.ts",
-      "import": "./dist/core-server/index.js",
       "default": "./dist/core-server/index.js"
     },
-    "./internal/core-server/presets/common-manager": {
-      "import": "./dist/core-server/presets/common-manager.js"
-    },
```

Comment:
How come these are no longer needed?

Reply:
it was never needed, it's only used by our common preset internally in core, so we can just reference it with the path instead of via an export map - which is what we already did.

https://github.com/storybookjs/storybook/pull/31825/files#diff-6f9af3b0651245cd00c4c837f2a699d9a14353d6c8f8281432f658e518475288R313-R316